tess-two/jni/com_googlecode_tesseract_android/src/android/jni/Application.mk

```diff
@@ -9,5 +9,5 @@ endif
 # Specify the hash namespace that we're using, based on the APP_STL we're using.
 APP_CFLAGS += -Werror -DHASH_NAMESPACE=__gnu_cxx -Wno-error=deprecated-register
 APP_PLATFORM := android-16
-APP_STL := gnustl_static
+APP_STL := c++_static
```

Comment:
This file is kind of a red herring and is ignored by this project, so the change on this line can be reverted. This file exists in the Tesseract code base and seems to be used for an internal build at Google.